src/core/Table/hooks/useSelectionCell.tsx

```diff
@@ -16,53 +13,9 @@ export const useSelectionCell = <T extends Record<string, unknown>>(
     return;
   }
 
-  hooks.allColumns.push((columns: ColumnInstance<T>[]) => [
-    {
-      id: SELECTION_CELL_ID,
-      disableResizing: true,
-      disableGroupBy: true,
-      minWidth: 48,
-      width: 48,
-      maxWidth: 48,
-      columnClassName: 'iui-slot',
-      cellClassName: 'iui-slot',
-      disableReordering: true,
-      Header: ({
-        getToggleAllRowsSelectedProps,
-        rows,
-        initialRows,
-        state,
-      }: HeaderProps<T>) => {
-        const disabled = rows.every((row) => isRowDisabled?.(row.original));
-        const checked = initialRows.every(
-          (row) =>
-            state.selectedRowIds[row.id] || isRowDisabled?.(row.original),
-        );
-        return (
-          <Checkbox
-            {...getToggleAllRowsSelectedProps()}
-            checked={checked && !disabled}
-            indeterminate={
-              !checked && Object.keys(state.selectedRowIds).length > 0
-            }
-            disabled={disabled}
-          />
-        );
-      },
-      Cell: ({ row }: CellProps<T>) => (
-        <Checkbox
-          {...row.getToggleRowSelectedProps()}
-          disabled={isRowDisabled?.(row.original)}
-          onClick={(e) => e.stopPropagation()} // Prevents triggering on row click
-        />
-      ),
-    },
-    ...columns,
-  ]);
-
-  hooks.useInstanceBeforeDimensions.push(({ headerGroups }) => {
```

Comment:
We don't need this fix anymore?

Reply:
I really hope not 😀 Because it was working ok when tested